tox.ini

```diff
@@ -3,10 +3,17 @@ envlist=bluecheck,doc8,flake8,rstcheck,docs,py36,py37,py38,py39
 skip_missing_interpreters=True
 
 [testenv]
-deps=pytest
 commands=pytest
+deps=
+    pytest
+    pytest-cov
 
 [pytest]
+addopts=
```

Comment:
I know you have a preference to add these configs to the `tox.ini` and that's totally fine.  FWIW, I usually add these to `setup.cfg`, e.g.:

```
[coverage:report]
fail_under = 100
show_missing = true

[coverage:run]
branch = true
parallel = true
omit =

[coverage:paths]
source =
    flufl/lock
```